.cg-docs/solutions/testing-patterns/2026-05-28-expect-warning-without-expect-error-swallows-error.md

```
@@ -0,0 +1,89 @@
+A test case that tested deprecation warnings passed interactively (where
+`pipfun::setup_working_release()` had been called) but failed during
+`R CMD check` with an unexpected error:
+
+```r
+# Before — fails under R CMD check
+pip_load_aux(measure = "ppp", file_to_load = "ppp_vintage") |>
+  expect_warning()
+```
+
+`R CMD check` error:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)